### ISSUE TITLE
Add 'xf86-input-synaptics' package into live system

### ISSUE THREAD

**Issue by notuxius** [netrunner-rolling/QA-Suggestions#135]
Supply this package along with 'xf86-input-libinput' package until all options will be available in 'Touchpad' KCM through libinput.

Currently most of touchpad settings are disabled if 'xf86-input-synaptics' is not installed in system.

with 'xf86-input-synaptics' package installed:
![touchpad-kcm-with-synaptics-package-installed](https://user-images.githubusercontent.com/2657014/29465167-fa2e89f4-8440-11e7-8cc2-e1869e81ad75.png)
![touchpad-kcm-with-synaptics-package-installed-2](https://user-images.githubusercontent.com/2657014/29465168-fa2f6608-8440-11e7-8c26-f37ea8823fc6.png)

and without:
![touchpad-kcm-without-synaptics-package-installed](https://user-images.githubusercontent.com/2657014/29465179-03a3c620-8441-11e7-9dd1-cbe0bedc3035.png)
![touchpad-kcm-without-synaptics-package-installed-2](https://user-images.githubusercontent.com/2657014/29465180-03a705a6-8441-11e7-955f-07217c9a5692.png)

**Comment by NuLogicSystems** [comment]
When the ISO was released this driver conflicted with xf86-input-libinput and could not be installed.

This package is also no longer being maintained (updated) upstream:
https://wiki.archlinux.org/index.php/Touchpad_Synaptics

As it will no longer be compatible with future versions of xorg-server (requires X-ABI-XINPUT_VERSION=24.1).

**Comment by NuLogicSystems** [comment]
Libinput supports most if not all of those features, plus some.
Libinput is not the issue, the kcm needs extending:
https://wiki.archlinux.org/index.php/Libinput#Touchpad_settings_not_taking_effect_in_KDE.27s_Touchpad_KCM

Have you also tried this yet? 
https://aur.archlinux.org/packages/kcm-pointing-devices-git/
https://wiki.archlinux.org/index.php/Libinput#Graphical_tools

**Comment by notuxius** [comment]
Yeah, just tried it. It has basic features comparing to synaptics package. I'm suggesting to adding synaptics package for convenient adjusting of touchpad all the same - until libinput will support most of features or until X11 package will complain about incompatibility as you say. What do you think? + pointing-devices-kcm.git repo is two years old.

**Comment by NuLogicSystems** [comment]
Once again, It's not libinput that is lacking, it's the KCM that needs further extending for libinput.
You can still adjust all these things with xinput set-prop, but I do get your point.

**Comment by notuxius** [comment]
+ increase default min/max and acceleration speed for touchpad. Currently it's to low comparing to mouse movement speed.

**Comment by NuLogicSystems** [comment]
Where is it slow? If you mean the in the dashboard menu, wallpapers, or other Plasma 5 components, these are known upstream issues with qtquickcontrol and libinput, not the drivers themselves. We ship imwheel to try and compensate for these mouse wheel issues. If your having the same problems with the synaptics driver and your touchpad, then please report this upstream so that the effected plasmoids can be updated to qtquickcontrols2 a bit faster.
